doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -408,33 +396,21 @@ Field         | Type       | Meaning
 `duration_ns` | [`bigint`] | The upper bound of the duration bucket in nanoseconds.
 `count`       | [`bigint`] | The number of recordings in the bucket.
 
-### `mz_compute_operator_durations`
-
-The `mz_compute_operator_durations` source stores a histogram describing the
-duration (an an interval) of each invocation for each [dataflow] operator.
+### `mz_scheduling_parks_histogram`
 
-Field         | Type          | Meaning
---------------|---------------|--------
-`id`          | [`bigint`]    | The ID of the operator. Corresponds to [`mz_dataflow_operators.id`](#mz_dataflow_operators).
-`worker_id`   | [`bigint`]    | The ID of the worker thread hosting the operator.
-`duration`    | [`interval`]  | The upper bound of the duration bucket as an interval.
-`count`       | [`bigint`]    | The number of recordings in the bucket.
-
-### `mz_scheduling_parks`
-
-The `mz_scheduling_parks` source stores a histogram describing [dataflow] worker
+The `mz_scheduling_parks_histogram` view describes a histogram describing [dataflow] worker
 park events. A park event occurs when a worker has no outstanding work.
 
-Field       | Type       | Meaning
-------------|------------| -------
-`worker_id` | [`bigint`] | The ID of the worker thread.
-`slept_for` | [`bigint`] | The actual length of the park event.
-`requested` | [`bigint`] | The requested length of the park event.
-`count`     | [`bigint`] | The number of park events in this bucket.
+Field          | Type       | Meaning
+---------------|------------| -------
+`worker_id`    | [`bigint`] | The ID of the worker thread.
+`slept_for_ns` | [`bigint`] | The actual length of the park event in nanoseconds.
+`requested_ns` | [`bigint`] | The requested length of the park event in nanoseconds.
+`count`        | [`bigint`] | The number of park events in this bucket.
 
-### `mz_raw_worker_compute_delays`
+### `mz_worker_compute_delays_histogram`
 
-The `mz_raw_worker_compute_delays` source provides, for each worker,
+The `mz_worker_compute_delays_histogram` view describes, for each worker,
```

Comment:
we say "for each worker" here, but we don't in (at least some of) the other per-worker relations.